.travis.yml

```diff
@@ -55,7 +55,7 @@ before_script:
       export _PYTEST_TOX_EXTRA_DEP=coverage-enable-subprocess
     fi
 
-script: tox -- testing/acceptance_test.py
+script: tox -- testing/test_capture.py
```

Comment:
In my manual testing it needs more, but `-m "not slow"` should do IIRC.

Reply:
Should be rebased for that though I guess.

Reply:
Trying `testing/python` now

But what do you suggest, `testing -m "not slow"`?

Reply:
Yes.. just `-m "not slow"` basically.